app/scripts/proactive/view/LoginView.js

```diff
@@ -210,10 +220,18 @@ define(
                     domainOptionString = domainOptionString + '</select></div>';
                     var domainOption = $(domainOptionString);
                     $("#login-basic").prepend(domainOption);
+
+                    if (domainName && domains.includes(domainName)) {
```

Comment:
don't you need to check if domains is not null ?

Reply:
It can't `var domainName = '';`. 
Actually I could have put only `domains.includes(domainName)`

Reply:
You mean it can't be null and domainName is not empty string ?